### PR TITLE
Rework tutorial part 2

### PR DESCRIPTION
**Pull request by jh-RLI**
- Rework tutorial part 2
-  increase usabalility for newcomers

### CODE REVIEW

**Comment by Bachibouzouk** [comment]
@jh-RLI - tell me when you want a review for this PR :)

**Comment by jh-RLI** [comment]
@Bachibouzouk you can add your review :)

File: api/OEP_API_tutorial_part2.ipynb [hunk 1]
```diff
@@ -136,17 +137,19 @@
...
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
+    "# insert 'columnname=value'\n",
```
Comment (Bachibouzouk) [inline api/OEP_API_tutorial_part2.ipynb]: Is it a comment, or commented code? If it is commented code it should not be pushed.

Comment (jh-RLI) [reply api/OEP_API_tutorial_part2.ipynb]: This is an example of what values should be used there

Comment (Bachibouzouk) [reply api/OEP_API_tutorial_part2.ipynb]: Ok :)

**Pull request merged**